### PR TITLE
Erik Winkel

### PR DESCRIPTION
**Pull request by erikwinkel**
**Deliverable Submission**

Please describe your comfort and completeness levels before submitting.

Comfort Level (1-5): 4

Completeness Level (1-5):4

What did you think of this deliverable?: Not too bad, but I couldn't have done it without looking at the past couple deliverables. I really need to work on making smaller, more frequent commits.

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -1,27 +1,50 @@
...
+      res.render('results', { searchResults: searchResults, search: req.query.q })
+    })
+    .catch(err => console.log(err))
+})
+
+// movies/:movie_id route - shows details of the movie
+app.get('/movies/:movie_id' , (req,res) => {
+  axios.get(`http://www.omdbapi.com/?i=${req.params.movie_id}&apikey=${omdbApiKey}`)
+    .then((response) => {
+      res.render('detail', {details: response.data, keys: Object.keys(response.data)})
```
Comment (NickSchmitt) [inline server.js:41]: What was the thought with `Object.keys` here?

File: views/detail.ejs [hunk 2]
```diff
@@ -1,2 +1,31 @@
 <% // Render individual movie details here %>
 <% // This is how to do comments in EJS, by the way... %>
+<% let stars = '' %>
+<% for(let i = Math.floor(details.imdbRating); i > 0; i--) {%>
```
Comment (NickSchmitt) [inline views/detail.ejs:4]: Nice

File: server.js (hunk 1 above)
Comment (erikwinkel) [reply server.js:41]: I forgot to take this out. My first version of detail.ejs just listed every key and it's value. Something like
<% keys.forEach(key => { %>
 <p> <%= key %>: <%= details[key] %> 
<% } %>

Comment (NickSchmitt) [reply server.js:41]: Cool, that does sound like a good case for a `for detail in details` loop over the `response.data` object

**Pull request closed without merging**